prod/westeurope/siem/event_hub/terragrunt.hcl

```diff
@@ -0,0 +1,25 @@
+dependency "resource_group_siem" {
+  config_path = "../resource_group"
+
+  mock_outputs = {
+    resource_group_name = "fixture"
+  }
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_eventhub?ref=v0.0.27"
+}
+
+inputs = {
+  name = "siem"
+  resource_group_name = dependency.resource_group_siem.outputs.resource_name
+  environment_short   = "s"
```

Comment:
Is it right to override the environment_short even so we are in prod environment?